Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -237,7 +245,14 @@ - (void)authorizationController:(ASAuthorizationController *)controller didCompl
         }
     }
     if ([platformAllowedCredentials count] || ![crossPlatformAllowedCredentials count]) {
-        auto request = adoptNS([[allocASAuthorizationPlatformPublicKeyCredentialProviderInstance() initWithRelyingPartyIdentifier:options.rpId] createCredentialAssertionRequestWithChallenge:toNSData(options.challenge).get()]);
+        RetainPtr<ASAuthorizationPlatformPublicKeyCredentialAssertionRequest> request;
+        ASAuthorizationPlatformPublicKeyCredentialProvider *provider = [allocASAuthorizationPlatformPublicKeyCredentialProviderInstance() initWithRelyingPartyIdentifier:options.rpId];
```

Comment:
Should use a RetainPtr and a adoptNS() to avoid leaking.

Reply:
Done